source/Preferences.h

```diff
@@ -147,6 +153,11 @@ class Preferences {
 	static BackgroundParallax GetBackgroundParallax();
 	static const std::string &ParallaxSetting();
 
+	// Extended jump effects setting, either "medium", "heavy", or "off".
```

Comment:
```suggestion
	// Extended jump effects setting, either "off", "medium", or "heavy".
```
to match the order - caus off is the default (like proximity is below)